extensions/api/util.py

```diff
@@ -75,6 +75,8 @@ def build_parameters(body, chat=False):
         'silero_tts_voice_pitch': body.get('silero_tts_voice_pitch', 'medium'),
         'silero_tts_voice_speed': body.get('silero_tts_voice_speed', 'medium'),
 
+        'eleven_labs_speaker' : body.get('el_speaker', 'Rachel'),
```

Comment:
rename to elevenlabs_speaker

As previously mentioned. Add a elevenlabs_enabled